src/BridgeController.js

```diff
@@ -0,0 +1,83 @@
+const InputView = require("./InputView");
+const OutputView = require("./OutputView");
+const BridgeGame = require("./BridgeGame");
+const BridgeMaker = require("./BridgeMaker");
+const BridgeRandomNumberGenerator = require("./BridgeRandomNumberGenerator");
+
+class BridgeController {
+  bridge;
+  tries = 1;
+
+  constructor() {
+    this.bridgeGame = new BridgeGame();
+  }
+
+  play() {
+    this.printBridgeGameStart();
+  }
+
+  printBridgeGameStart() {
+    OutputView.printStart();
+    this.getBridge();
+  }
+
+  getBridge() {
+    InputView.readBridgeSize(
+      this.makeBridge.bind(this),
+      this.startBridge.bind(this)
+    );
+  }
```

Comment:
```suggestion
  getBridge(..callbacks){
    InputView.readBridgeSize(...callbacks);
  }
```
이렇게 외부에서 받아주면 좋을 것 같아요!:)